tests/test_multiplatform.py

```diff
@@ -0,0 +1,410 @@
+import os
+from typing import List
+from unittest.mock import patch
+
+import pytest
+from python_on_whales import docker
+from python_on_whales.exceptions import DockerException
+
+from buildrunner.docker.multiplatform_image_builder import (
+    ImageInfo, MultiplatformImageBuilder)
+
+TEST_DIR = os.path.basename(os.path.dirname(__file__))
+
+# FIXME: These tests can be broken if a custom buildx builder is set as default  # pylint: disable=fixme
+
+def actual_images_match_expected(actual_images, expected_images) -> List[str]:
```

Comment:
Should have two lines in between methods/declarations.

Reply:
done